malaba6/E-Rental
# #167147965 built sign-in page

#### What does this PR do?
It creates a sign-in page.

#### Description of Task to be completed?
1. It creates and adds a sign-in page to the sign-in button on the homepage.
2. It adjusts some HTML and CSS of the homepage to maintain consistency.

#### How should this be manually tested?
Navigate to the homepage page and click on the **Sign In** button and check if you get the to the login form. 

#### Secreenshots

**Desktop Version**

![sign-in html](https://user-images.githubusercontent.com/34879554/61404672-41b08300-a8d8-11e9-8c79-deb9e47e26f3.png)


**Mobile Version**

![sign-in html-mobile](https://user-images.githubusercontent.com/34879554/61404691-49702780-a8d8-11e9-898d-52b67f713abb.png)
